functions.go

```diff
@@ -101,6 +102,11 @@ func unpackServiceURLArgs(args []string) (string, string, string, string, string
 	return args[0], args[1], args[2], args[3], args[4], args[5]
 }
 
+func checkIfServiceInLocal(serviceName string, localDevList string) bool {
+	matched, _ := regexp.MatchString(`(,|^)\s*` + serviceName + `\s*(,|$)`, localDevList)
```

Comment:
**suggestion:** Consider breaking the `localDevList` into a Slice of strings and then check for a match?

I'm not sure what that would look like in Go, but we'd avoid using regular expressions and would be instead checking presence of a string in a list, which feels closer to our intention.

The regular expression works just fine though, definitely consider this optional.